babel/src/main/java/org/apache/calcite/sql/babel/SqlAlterTable.java

```diff
@@ -34,18 +34,22 @@
 public class SqlAlterTable extends SqlAlter {
   final SqlIdentifier tableName;
```

Comment:
nit: I am not sure if the default access specifier is appropriate for these final variable. You might consider using "private".

Reply:
Ok, I will mark them public, since this is what has been done in [similar classes in babel](https://github.com/googleinterns/calcite/blob/630d38ea02f57847699a2f2edde85566679a47b8/babel/src/main/java/org/apache/calcite/sql/babel/SqlCreateTable.java#L41).